website/src/components/Account/AccountModule.js

```diff
@@ -0,0 +1,249 @@
+import React, { useCallback, useEffect, useMemo, useRef, useState } from 'react'
+import { useWallet } from 'use-wallet'
+import { Button, GU, IconConnect, springs, useViewport } from '@aragon/ui'
+import { Transition, animated } from 'react-spring/renderprops'
+import { shortenAddress, getUseWalletProviders } from '../../utils/web3-utils'
+import AccountButton from './AccountButton'
+import HeaderPopover from './HeaderPopover'
+import ScreenConnected from './ScreenConnected'
+import ScreenConnecting from './ScreenConnecting'
+import ScreenError from './ScreenError'
+import ScreenProviders from './ScreenProviders'
+
+const AnimDiv = animated.div
+
+const SCREENS = [
+  {
```

Comment:
Can you condense this array a bit more? I know its kind of a nit, but usually array of objects in the code base are written as

```js
const something = [{
  foo: 'bar'
}, {
  bar: 'baz'
}]
```

Reply:
Done in https://github.com/1Hive/apiary/pull/181/commits/9fc22511a86e8d560ecf888b640de76873a24829.